shared/domains/chelonia/internals.js

```diff
@@ -1144,6 +1146,12 @@ export default (sbp('sbp/selectors/register', {
       this.config.reactiveSet(state[contractID], '_volatile', currentVolatileState)
     }
     const { HEAD: latestHEAD } = await sbp('chelonia/out/latestHEADInfo', contractID)
+      .catch(e => {
+        console.error(`[chelonia] chelonia/out/latestHEADInfo error: ${e.message || e}`, e)
+        emitContractSyncStop()
+        throw e
+      })
```

Comment:
There's a simpler solution: if we move the `try` block near the top of this function (just after `const state = ...`, then that's all that's needed and the rest of the function can be unmodified.

Reply:
right. done it.